shared/js/loadApp.js

```diff
@@ -29,7 +29,7 @@ module.exports = function (callback) {
     }
   };
 
-  if (!appOptions.channel) {
+  if (!appOptions.channel && dashboard.clientState.queryParams('solution') !== 'true') {
```

Comment:
Nit: Would be nice to extract this query param check into a descriptive helper (e.g. `... && !viewingSolution()`).